spec/controllers/meetings_controller_spec.rb

```diff
@@ -149,11 +149,12 @@
       expect(JSON.parse(response.body)['data']).to eq({ 'joinUrl' => 'JOIN_URL', 'status' => true })
     end
 
-    it 'returns status false if the meeting is NOT running' do
+    it 'returns status false if the meeting is NOT running and the user is NOT authorized to start the meeting' do
       allow_any_instance_of(BigBlueButtonApi).to receive(:meeting_running?).and_return(false)
       expect_any_instance_of(BigBlueButtonApi).not_to receive(:join_meeting)
 
-      post :status, params: { friendly_id: room.friendly_id, name: user.name }
+      post :status, params: { friendly_id: test_room.friendly_id, name: user.name }
```

Comment:
I believe that #status should never return false to a Moderator as hitting the #status endpoint as a moderator/room owner will start the meeting.

Reply:
I was confused, this test should not have been passing - it is because the #status endpoint should never be hit by a room owner because the RoomJoin page is not available to room owners.